src/lib/shell/commands/Dns.cpp

```diff
@@ -42,23 +42,23 @@ class DnsShellResolverDelegate : public Dnssd::ResolverDelegate
 public:
     void OnNodeIdResolved(const Dnssd::ResolvedNodeData & nodeData) override
     {
-        streamer_printf(streamer_get(), "DNS resolve for " ChipLogFormatX64 "-" ChipLogFormatX64 " succeeded:\n",
+        streamer_printf(streamer_get(), "DNS resolve for " ChipLogFormatX64 "-" ChipLogFormatX64 " succeeded:\r\n",
```

Comment:
why did you add \r to shared code? Is it required by the interface? Could it be  resolved at a lower level only for platforms that need it? I am used to printfs only having the '\n' and  woner why  the \r is needed here

Reply:
"\r" was added because some terminals are not adding it by default. It shouldn't negatively impact other platforms.

Reply:
offline discussion: uart in macos apparently also wants \r.